### DISCUSSION TITLE
missing notebook in template

### DISCUSSION THREAD

**Discussion by Joerg0** [sebastian-hofstaetter/teaching#131] [Advanced IR: Exercise Q&A] [answered]
In the asignment_2.md file is written:
* You should use an Nvidia GPU, if you don't have one locally, you can use Google Colab (https://colab.research.google.com/), which offers a free GPU in the cloud -> use the train.ipynb for this instead of train.py

-> There is not train.ipynb in our template. Where do I get it from?

**Comment by whoopi24** [comment, marked as answer]
Hi, 
I think that the mentioned train.py file is actually the re_ranking.py file from the template - so just a renaming mistake in the assignment description (hopefully) and not a missing Python file.

Best regards,
Marina

**Comment by Joerg0** [comment]
Ah, yes, probably! thx
